docs/recipes/circleci-workflows.md

```diff
@@ -1 +1,75 @@
-# CircleCI 2.0 workflows
+# Using semantic-release with [CircleCI 2.0 workflows](https://circleci.com/docs/2.0/workflows/)
+
+![CircleCI workflows screenshot](https://github.com/circleci/circleci-docs/raw/928cbb042453ae182996ce05f4e4042b02a24634/jekyll/assets/img/docs/workflow_detail.png)
+
+Similar to Travis build stages, CircleCI 2.0 workflows are a great way to run semantic-release in an independent build step.
+We can let CircleCI take care of only running a release on master, after tests passed and running it on an image with Node 8 - your test jobs could use a completely different environment, your project may not even be written in JavaScript.
+
+Make sure to set `NPM_TOKEN` and `GH_TOKEN` in Circle's project settings UI.
```

Comment:
Can you reference the CircleCi documentation to do that? Similarly to [travis.md#environment-variables](https://github.com/semantic-release/semantic-release/blob/caribou/docs/recipes/travis.md#environment-variables).